testing/files/entrypoint.sh

```diff
@@ -74,7 +75,9 @@ if [[ $overwrite == "y" || $overwrite == "Y" ]]; then
 		./build/bin/beacond genesis add-premined-deposit --home $HOMEDIR \
 			32000000000 0x20f33ce90a13a4b5e7697e3544c3083b8f8a51d4
 		./build/bin/beacond genesis collect-premined-deposits --home $HOMEDIR
-		./build/bin/beacond genesis execution-payload "$ETH_GENESIS" --home $HOMEDIR
+		./build/bin/beacond genesis set-deposit-storage "$ETH_GENESIS" --home $HOMEDIR
```

Comment:
@nidhi-singh02 why do we execute both commands?

Reply:
As we want to modify the deposit contract storage slot values for both the genesis files - for nethermind and for others. @calbera I hope my understanding is correct.

Reply:
Yeah the idea here is set both eth genesis files so that in case the user wants to start nethermind as the EL, its genesis file is also ready